src/Symfony/Bundle/FrameworkBundle/CacheWarmer/TranslationsCacheWarmer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\CacheWarmer;
+
+use Symfony\Component\HttpKernel\CacheWarmer\CacheWarmerInterface;
+use Symfony\Component\HttpKernel\CacheWarmer\WarmableInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Generates the catalogues for translations.
+ *
+ * @author Xavier Leune <xavier.leune@gmail.com>
+ */
+class TranslationsCacheWarmer implements CacheWarmerInterface
+{
+    protected $translator;
+
+    /**
+     * Constructor.
+     *
+     * @param TranslatorInterface $translator A Translator instance
+     */
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * Warms up the cache.
+     *
+     * @param string $cacheDir The cache directory
+     */
```

Comment:
No need to duplicate the docblock here. Just use (same with the `warmUp()` method):

```php
/**
 * {@inheritdoc}
 */
```